roles/self_hosted_first_host/defaults/main.yml

```diff
@@ -30,9 +30,17 @@ package_name: "{{ 'rhevm-appliance' if compatibility_version < 4.1 else 'rhvm-ap
 
 # user-data
 engine_db_host: localhost
-engine_db_user: root
+engine_db_user: engine
+engine_db_name: engine
 engine_db_port: 5432
 engine_db_password: dbpassword
 engine_organization: localdomain
 engine_application_mode: virt
 ssh_authorized_keys: []
+
+# set dwh default values
+ovirt_engine_dwh_db_host: localhost
```

Comment:
Could you either make this `engine_dwh_db_host` or make the above `engine_db_host` `ovirt_engine_db_host`, just for consistency?